src/index.js

```diff
@@ -39,4 +46,60 @@ myPromise.then((result) => {
         <p class="reservations">Reservations</p>  
         </div>`;
   });
+  const commentbutton = document.querySelectorAll('.comments');
```

Comment:
1.Instead of using querySelectorAll, use getElementById to select elements by ID. IDs are unique, so using getElementById is faster and more efficient.
`javascript
const commentButton = document.getElementById('comments');`